config/initializers/rack_attack.rb

```diff
@@ -4,7 +4,7 @@ class Attack
     # Throttle all requests by IP (60rpm)
```

Comment:
Oh this should be 100 then

```suggestion
    # Throttle all requests by IP (100rpm)
```